src/App.tsx

```diff
@@ -1,27 +1,116 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classnames from 'classnames';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import 'bulma';
+import { getAll, get5First, getRed } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goodsAll, setGoodsAll] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const [loadingAll, setLoadingAll] = useState(false);
```

Comment:
It's not required in this task
But if you want to show `loader` create an enum with `all`, `five`, and `red` values, and set `null` or enum value in the state